src/rgw/setup/rgw_service

```diff
@@ -37,9 +37,9 @@ while [ $# -gt 0 ];  do
     shift 1
 done
 
-function start_rgw {
+function start_service {
```

Comment:
again, after PR #54 , this file wont be there. rebase after that is merged and then take this forward

Reply:
done